core/domain/user_services.py

```diff
@@ -69,18 +72,22 @@ class UserSettings(object):
         preferred_site_language_code: str or None. System language preference.
     """
     def __init__(
-            self, user_id, email, username=None, last_agreed_to_terms=None,
+            self, user_id, email, username=None,
+            role=feconf.ROLE_EXPLORATION_EDITOR, last_agreed_to_terms=None,
```

Comment:
done. I have made changes accordingly. 
But, I noticed that in many places in tests both `signup` and `get_or_create_user` function is used. Why?